### ISSUE TITLE
[FEATURE REQ] Add keybinding to kill processes

### ISSUE THREAD

**Issue by anihm136** [pesos/grofer#16]
Generally I use a system profiler to monitor processes taking up large amounts of system resources, which I would usually need to kill. Having a keybinding to kill a process from the proc page would be very helpful in this regard

**Describe the solution you'd like**
A simple single key keybinding to kill a highlighted process from within the proc page

**Describe alternatives you've considered if any**
None

**Comment by MadhavJivrajani** [comment]
Let's work on this issue once #31 is done.

**Comment by anihm136** [comment]
Okay makes sense

**Comment by anihm136** [comment]
What's the status on this now that the termdash migration is on hold?

**Comment by MadhavJivrajani** [comment]
I think we can continue work on this, looks like we'll be using termui for now at least

**Comment by MadhavJivrajani** [comment]
References to solve this issue: 

- To get the selected row in the UI for `grofer proc` command, the `SelectedRow` field of the `List` type can be user ([link](https://github.com/gizak/termui/blob/4cca61d83fa2cc0f485c478ff768b0108f6591d6/v3/widgets/list.go#L20))
- To send a kill signal to the process the [Kill](https://godoc.org/github.com/shirou/gopsutil/process#Process.Kill) of the gopsutil can be used.

**Comment by Samyak2** [comment]
Can I work on this issue?

**Comment by MadhavJivrajani** [comment]
Sure!
